cmd/earth/main.go

```diff
@@ -972,6 +975,11 @@ func (app *earthApp) run(ctx context.Context, args []string) int {
 				"Check your git auth settings.\n" +
 					"Did you ssh-add today? Need to configure ~/.earthly/config.yml?\n" +
 					"For more information see https://docs.earthly.dev/guides/auth\n")
+		} else if rpcRegex.MatchString(err.Error()) {
+			baseErr := errors.Cause(err)
+			baseErrMsg := rpcRegex.ReplaceAll([]byte(baseErr.Error()), []byte(""))
+
+			app.console.Warnf("Error: %v\n", string(baseErrMsg))
 		} else {
 			app.console.Warnf("Error: %v\n", err)
```

Comment:
I see maybe we could also do `errors.Cause(err)` here.

Reply:
That could be a good idea, if `--verbose` is not passed